README.rst

```diff
@@ -1,9 +1,67 @@
-========
-Overview
-========
+================
+pytest-openfiles
+================
 
-This plugin detects whether any file handles or other file-like objects were
-inadvertently left open at the end of a test. It has been moved from the core
-astropy project since it is of use more generally. It is currently WIP.
+This package provides a plugin for the `pytest`_ framework that allows
+developers to detect whether any file handles or other file-like objects were
+inadvertently left open at the end of a unit test. It has been moved from the
+core `astropy`_ project since it is of use more generally.
 
-This plugin is not supported with Py2.7 on Win32 platforms.
+.. _pytest: https://pytest.org/en/latest/
+.. _astropy: https://astropy.org/en/latest/
+
+**NOTE**: This plugin is not supported with Py2.7 on Win32 platforms.
```

Comment:
if we are to backport pytest-astropy to astropy v2.0.x, would this be an issue or then just simply this plugin won't be installed and thus we can easily pull the version from astropy rather?

Reply:
I'm hoping that it will work the way you describe. Even if it doesn't, I'm not sure whether it will be a problem since I don't think this plugin ever worked on Py2.7 on Win32, but astropy probably never tested it there.

Reply:
ohhh, ok, I misread it and assumed it's python2 or win32 (take away, never to do review during a conference? :) )

Reply:
What is so specific; How did you find out? Is Appveyor 64-bit only?

Reply:
I'm not sure what the exact issue is, but when I originally set up the appveyor test matrix I tried to make it as comprehensive as possible, but the Py27+Win32 tests failed. It's not worth supporting, so we won't support it.

Reply:
well, the only good reason I heard about to still support 32bit is chandra, but they fortunately don't run windows, so this totally looks fine.